packages/ingestor/src/lib/skylark/saas/create.ts

```diff
@@ -0,0 +1,360 @@
+import { FieldSet, Records } from "airtable";
+import { jsonToGraphQLQuery } from "json-to-graphql-query";
+import { chunk, flatten, has, isString, values } from "lodash";
+import { GraphQLBaseObject, GraphQLMetadata } from "../../interfaces";
+import {
+  ApiObjectType,
+  MediaObjectTypes,
+  GraphQLObjectTypes,
+  RelationshipsLink,
+  ValidMediaObjectRelationships,
+} from "../../types";
+import { getValidPropertiesForObject, getExistingObjects } from "./get";
+import { graphQLClient } from "./graphql";
+import {
+  getExtId,
+  gqlObjectMeta,
+  getUidsFromField,
+  getValidFields,
+} from "./utils";
+
+const createOrUpdateMultipleObjects = async (
+  name: string,
+  mutations: { [key: string]: object }
+): Promise<GraphQLBaseObject[]> => {
+  // Smaller requests are better as each is handled by a single lambda
+  const chunks = chunk(Object.keys(mutations), 20);
+
+  const chunkedData = await Promise.all(
+    chunks.map(async (keys, i): Promise<GraphQLBaseObject[]> => {
+      const splitMutations = keys.reduce(
+        (previousObj, key) => ({
+          ...previousObj,
+          [key]: mutations[key],
+        }),
+        {}
+      );
+
+      const mutation = {
+        mutation: {
+          __name: chunks.length > 1 ? `${name}_chunk_${i + 1}` : name,
+          ...splitMutations,
+        },
+      };
+
+      const graphQLMutation = jsonToGraphQLQuery(mutation);
+
+      const data = await graphQLClient.request<{
+        [key: string]: GraphQLBaseObject;
+      }>(graphQLMutation);
+
+      const arr = values(data);
```

Comment:
this `const` is needed? can we return `values(data)` ?

Reply:
good for debug